<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_Vin1" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_V3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_Vout1" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_Vin2" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_Vout2" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>280</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="setpoint1">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>70</y>
      <width>29</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="setpoint2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>310</y>
      <width>29</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>customPlot</zorder>
   <zorder>customPlot_Vin1</zorder>
   <zorder>customPlot_V3</zorder>
   <zorder>widget_3</zorder>
   <zorder>customPlot_Vout1</zorder>
   <zorder>customPlot_Vin2</zorder>
   <zorder>customPlot_Vout2</zorder>
   <zorder>setpoint1</zorder>
   <zorder>setpoint2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
